src/content/docs/docs/developers/getting-started/index.md

```diff
@@ -188,3 +188,4 @@ For production deployments, use the following Mainnet details. It is recommended
 | Boar | `https://rpc-http.mezo.boar.network` | `wss://rpc-ws.mezo.boar.network` |
 | Imperator | `https://rpc_evm-mezo.imperator.co` | `wss://ws_evm-mezo.imperator.co` |
 | Validation Cloud | `https://mainnet.mezo.public.validationcloud.io` | `wss://mainnet.mezo.public.validationcloud.io` |
+| dRPC NodeCloud | `https://mezo-testnet.drpc.org` | `wss://mezo-testnet.drpc.org` |
```

Comment:
These addresses indicate that these are testnets, but other RPC providers have mainnet addresses in this table. Also, if I'm not mistaken this table is under the point [5. Mainnet Network Details](https://mezo.org/docs/developers/getting-started)

Maybe better place to put it is under Mezo Testnet [here](https://mezo.org/docs/users/getting-started/connect).

And on a separate note, I haven't been involved in any of the conversations where we approved them to be an official testnet rpc provider, so I'm verifying just a "place" where we can put such addresses, not the actual urls / agreements with them.